### ISSUE TITLE
NTR: microbiome sequencing design

### ISSUE THREAD

**Issue by TrisSN** [obi-ontology/obi#990]
NTR: microbiome sequencing design
definition: The microbiome comprises all of the genetic material within a microbiota (the entire collection of microorganisms in a specific niche, such as the human gut). This can also be referred to as the metagenome of the microbiota. [Source: https://www.nature.com/subjects/microbiome]
Parent: study design OBI:0500000
synonyms: microbiota sequencing design, metagenome sequencing design

**Comment by jamesaoverton** [comment]
Thanks for the term request. We discussed this on the 2018-12-10 OBI call.

1. The definition you've provided is for 'microbiome' and not for any kind of study design. How would you define the 'microbiome sequencing design'?
2. 'microbiota sequencing design' and 'metagenome sequencing design' are quite different, and we might want to have more than one class here.
3. How does your proposed term compare to our existing 'taxonomic diversity assessment by targeted gene survey' http://purl.obolibrary.org/obo/OBI_0001960 ?

**Comment by TrisSN** [comment]
Hi, Sorry for being slow to get back to you on this.
1) Yes, I couldn't find a design definition online, so went with just the microbiome. If I can provide the definition myself, following a typical OBI study design example: "microbiome sequencing design is a study design in which all of the genetic material within a microbiota (the entire collection of microorganisms in a specific niche, such as the human gut) are sequenced."
2) Those synonyms I gave are perhaps not rigorous enough--I'm not specialist in this area, but I think the metagenome term is less appropriate here. I'll retract that one. microbiome vs microbiota... may be the same(?)
3) 'taxonomic diversity assessment by targeted gene survey' would actually be very useful to us if we could get it as a study design (we restrict ourselves to these for our Design Type metadata). As is, it refers to the actual assessment rather than the more abstract 'design' idea. 'taxonomic diversity assessment by targeted gene survey study design'? Too unwieldy?

**Comment by DanBerrios** [comment]
1. We definitely want to separate out terms/definitions for microbiome, microbiota, and microbiome study designs, as James pointed out
2. Not all genetic material within microbiota is sequenced, so need to adjust definition of microbiome study design accordingly.  Something like: "microbiome sequencing design is a study design in which  the genetic material within microbiota (the entire collection of microorganisms in a specific niche, such as the human gut) are sequenced"

**Comment by DanBerrios** [comment]
Consider whether this TR fits in with previously patterned terms: <study goal> by <study lab methods>

**Comment by TrisSN** [comment]
> 1. We definitely want to separate out terms/definitions for microbiome, microbiota, and microbiome study designs, as James pointed out
> 2. Not all genetic material within microbiota is sequenced, so need to adjust definition of microbiome study design accordingly.  Something like: "microbiome sequencing design is a study design in which  the genetic material within microbiota (the entire collection of microorganisms in a specific niche, such as the human gut) are sequenced"

Yeah, I agree with this definition.

**Comment by cmungall** [comment]
@ramonawalls should microbiome be in PCO or ecocore? cc @diatomsRcool @pbuttigieg

**Comment by zhengj2007** [comment]
'microbiome' has been defined in OHMI http://purl.obolibrary.org/obo/OHMI_0000003, as a subClassOf ENVO:biome.

**Comment by diatomsRcool** [comment]
Sounds like it already lives somewhere, but I'm ok with it being in ecocore. @ramonawalls we need to have that chat about what goes in ecocore vs PCO vs ENVO.

**Comment by cmungall** [comment]
Yes I think we need to think very carefully about modularity to avoid patching together hierarchies from multiple different ontologies

**Comment by ramonawalls** [comment]
Sorry for the delay -- I was out of town.  As pointed about above, we should carefully define classes for the community (in PCO), for the biomes in which they live (probably already covered in ENVO, but new classes could be added), and the study design requested here.

The OHMI definition is semantically messy, in that is confounds the biome and the microbial community. OHMI says that microbiome is a subclass of ENVO biome, but then it goes on to define it as "a complex mixture of microorganisms that reside in a specific environmental niche." That sounds more like a collection of organisms to me.  If you look at how OHMI defines and uses their microbiome classes (e.h., "host-microbiome interaction subClassOf : has participant some microbiome") they seems to be referring to the community, not the biome. People do in fact use the term "microbiome" to describe just the set of microrganisms living in a certain time or place, but then it should not be a subclass of biome (which is "an ecosystem to which resident ecological communities have evolved adaptations"). 

We already have a term in PCO for microbial community (http://www.ontobee.org/ontology/PCO?iri=http://purl.obolibrary.org/obo/PCO_1000004), and I suggest that be the term that is used to define the study design. Adding "microbiome" as a related synonym to PCO:1000004 would be useful.

As for the biomes OHMI terms, if they really want to describe the organismal community, they should request those classes in PCO. If they really want to describe the biomes, they should request new subclasses of ENVO: environmental system.  Where the interaction classes belong is a discussion well beyond the scope of this issue.

@pbuttigieg
